src/helpers/url.js

```diff
@@ -46,7 +46,7 @@ const getWopiUrl = ({ fileId, title, readOnly, closeButton, revisionHistory }) =
 	return Config.get('urlsrc')
 		+ 'WOPISrc=' + wopisrc
 		+ '&title=' + encodeURIComponent(title)
-		+ '&lang=' + languageToBCP47()
+		+ '&ui=' + getLocale().replace(/_/g, '-')
```

Comment:
As https://wopi.readthedocs.io/en/latest/discovery.html#term-ui-llcc defines the UI language it should use getLanguage() in this case.

https://wopi.readthedocs.io/en/latest/faq/languages.html#languages
Single language codes from Nextcloud like "en" should also be fine here according to the Office API docs: 

> You may also use any value provided it is in the format described in RFC 1766.